ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -0,0 +1,206 @@
+import React, { useCallback, useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import log from 'loglevel';
+import { EVENT } from '../../../../shared/constants/metametrics';
+import ActionableMessage from '../../ui/actionable-message';
+import Popover from '../../ui/popover';
+import Checkbox from '../../ui/check-box';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import { I18nContext } from '../../../contexts/i18n';
+import { PageContainerFooter } from '../../ui/page-container';
+import {
+  accountsWithSendEtherInfoSelector,
+  getSubjectMetadata,
+} from '../../../selectors';
+import { getAccountByAddress } from '../../../helpers/utils/util';
+import formatMessage from './format-message-params';
+import Header from './signature-request-siwe-header';
+import Message from './signature-request-siwe-message';
+
+export default function SignatureRequestSIWE({
+  txData,
+  cancelPersonalMessage,
+  signPersonalMessage,
+}) {
+  const allAccounts = useSelector(accountsWithSendEtherInfoSelector);
+  const subjectMetadata = useSelector(getSubjectMetadata);
+
+  const {
+    msgParams: {
+      from,
+      origin,
+      siwe: { parsedMessage },
+      version,
+    },
+    type,
+  } = txData;
+
+  const fromAccount = getAccountByAddress(allAccounts, from);
+  const targetSubjectMetadata = subjectMetadata[origin];
+
+  const t = useContext(I18nContext);
+  const trackEvent = useContext(MetaMetricsContext);
+
+  const isMatchingAddress = from === parsedMessage.address;
+
+  const checkSIWEDomain = () => {
+    let isSIWEDomainValid = false;
+
+    if (origin) {
+      const { host } = new URL(origin);
+      isSIWEDomainValid = parsedMessage.domain === host;
+    }
+    return isSIWEDomainValid;
+  };
+
+  const isSIWEDomainValid = checkSIWEDomain();
+
+  const [isShowingDomainWarning, setIsShowingDomainWarning] = useState(false);
+  const [agreeToDomainWarning, setAgreeToDomainWarning] = useState(false);
+
+  /**
+   * @todo update MetaMetrics for onSign and onCancel
+   * Currently, we're tracking the events similarily to how we track the events with the SignatureRequest component
+   * in which the action names differ. We want to revisit this to add properties mentioned in the related, internal Slack thread.
+   * We would also want to ensure that we are tracking the event appropriatedly should an error be thrown.
+   * @see {@link ui/components/app/signature-request/signature-request.component.js}
+   * @see {@link https://consensys.slack.com/archives/C031PSFHQER/p1654811895007879?thread_ts=1654805714.261689&cid=C031PSFHQER}
+   */
```

Comment:
In my opinion not tracking metrics is better than tracking them based on the outdated schema. We can revisit later. I have a pretty good idea on how we can track these in the future.

Reply:
@brad-decker heads up, SIWE reuses the same RPC method call name as `personal_sign` making our current tracking solution in createRPCMethodTrackingMiddleware.js unable to support SIWE out of the box. 

I'm okay with removing the current tracking metrics in this PR. It can almost always, if not always, be a good idea to separate metric logic into its own PR. I can also see the benefit of keeping these trackEvents in while we reprioritize and implement an updated version of the metrics here.

cc: @kevinghim

Reply:
we can add it using the detect method that is being added here, it will be trivial. I already scoped this out when I was re-doing #15082

Reply:
Removed all references to `trackEvent` in signature-request-siwe. We can add this back in via the improved metrics flow